src/clojure/nrepl/middleware/interruptible_eval.clj

```diff
@@ -117,8 +117,12 @@
                     (.flush ^Writer out)
                     (reset! session (maybe-restore-original-ns @bindings))
                     (t/send transport (response-for msg
-                                                    {:value v
-                                                     :ns (-> *ns* ns-name str)})))
+                                                    (merge
+                                                     {:value v
+                                                      :ns (-> *ns* ns-name str)}
+                                                     (if trim
```

Comment:
I think the `merge` and `if` are redundant. You can simply pass `trim` as nil if it's not in the request.